src/main/java/Storage.java

```diff
@@ -0,0 +1,81 @@
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+import static java.lang.System.lineSeparator;
+
+public class Storage {
```

Comment:
Consider having the methods not be `static`, reasons for which will be learnt later.